storage/store.go

```diff
@@ -1013,7 +1012,7 @@ func (s *Store) maybeResolveWriteIntentError(rng *Range, method string, args pro
 			Txn:          args.Header().Txn,
 		},
 		PusheeTxn: wiErr.Txn,
-		Abort:     proto.IsReadWrite(method), // abort if cmd is read/write
+		Abort:     proto.IsWrite(args), // abort if cmd is read/write
```

Comment:
The old code was using "read/write" imprecisely to mean "write", so the code is correct and the comment should be changed.